csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/NugetPackageRestorer.cs

```diff
@@ -674,10 +700,40 @@ private bool IsFeedReachable(string feed, int timeoutMilliSeconds, int tryCount,
             return (timeoutMilliSeconds, tryCount);
         }
 
+        /// <summary>
+        /// Checks that we can connect to all Nuget feeds that are explicitly configured in configuration files.
+        /// </summary>
+        /// <param name="explicitFeeds">Outputs the set of explicit feeds.</param>
+        /// <returns>True if all feeds are reachable or false otherwise.</returns>
         private bool CheckFeeds(out HashSet<string> explicitFeeds)
```

Comment:
Maybe rename one of the `CheckFeeds` methods. The naming is a bit confusing when their type signatures are so similar.

Reply:
Done in https://github.com/github/codeql/pull/18850/commits/d2b88ae5a86f5e2c369ebfd9e31198d1ca16f3c8